FirebaseAppDistribution/Sources/FIRAppDistributionAuthPersistence.m

```diff
@@ -111,14 +124,18 @@ + (BOOL)persistAuthState:(OIDAuthState *)authState error:(NSError **_Nullable)er
 
 + (NSMutableDictionary *)getKeyChainQuery {
   NSMutableDictionary *keychainQuery = [NSMutableDictionary
-      dictionaryWithObjectsAndKeys:(id)kSecClassGenericPassword, (id)kSecClass, @"OAuth",
-                                   (id)kSecAttrGeneric, @"OAuth", (id)kSecAttrAccount,
-                                   @"fire-fad-auth", (id)kSecAttrService,
-                                   (id)kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly,
-                                   (id)kSecAttrAccessible, nil];
+      dictionaryWithObjectsAndKeys:(id)kSecClassGenericPassword, (id)kSecClass,
+                                   kFIRAppDistributionAuthPersistenceErrorKeychainId,
+                                   (id)kSecAttrGeneric, [self bundleIdentifier],
+                                   (id)kSecAttrAccount, [self bundleIdentifier],
```

Comment:
I would suggest to add a Firebase specific part to the service name to prevent a potential collision for the host app or another SDK.

Reply:
Good idea. Should I use different values for account attribute and the service id attribute? Or would using the same ID's be okay?

Reply:
I don't think the same string for the account name and the service name is an issue. It should be okay.

Reply:
Added app id to the keychain key Id to prevent collision.